chart/templates/mayastor/agents/ha/ha-node-daemonset.yaml

```diff
@@ -70,6 +71,8 @@ spec:
           mountPath: /sys
         - name: run-udev
           mountPath: /run/udev
+        - name: plugin-dir
+          mountPath: /csi
```

Comment:
hmm is there any point for `Values.csi.node.socketPath` if `/csi` is still hardcoded? Or should we make `/csi` part of the variables as well and derive socketPath?

Reply:
Yeah, was also thinking of same. That would be better.

Reply:
Done.